packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts

```diff
@@ -1280,6 +1280,11 @@ type MessagesType = {
   'scenes.earninterest.earnupgrade.header': 'Put your crypto to work by upgrading to Gold Level.'
   'scenes.earninterest.form.earn3percent': 'Earn 3% AER on your BTC'
   'scenes.earninterest.form.earnbutton': 'Earn Interest'
+  'scenes.earninterest.history.amount': 'Amount'
+  'scenes.earninterest.history.date': 'Date'
+  'scenes.earninterest.history.from': 'From'
+  'scenes.earninterest.history.to': 'To'
+  'scenes.earninterest.history.type': 'Type'
```

Comment:
Quite a few instances of these strings already, can you create `copy.amount: Amount` and clean that up a little?